### DISCUSSION TITLE
Conflict: Multiple assets emit different content to the same filename index.html

### DISCUSSION THREAD

**Discussion by wengjiacheng** [web-infra-dev/rsbuild#1224] [Q&A]
我在使用rsbuild构建一个vue2的spa项目
在rsbuild.config.mjs中做添加了如下配置
`
output: {
    assetPrefix: "/cv3/",
    distPath: {
      root: "dist/cv3",
    },
    copy: [
      {
        from: "./public",
        to: "",
        toType: "dir",
        ignore: [".DS_Store", { glob: "index.html", matchBase: false }],
      },
    ],
  },
  source: {
    alias: {
      "@": "./src",
    },
  },
  plugins: [pluginVue2()],
  html: {
    template: "./public/index.html",
  },
  dev: {
    assetPrefix: "/cv3/",
  },
  tools: {
    bundlerChain: (chain, { CHAIN_ID }) => {
      chain.module
        .rule("style-resources-loader")
        .test(/\.less$/)
        .oneOf(CHAIN_ID.RULE.LESS)
        .before(CHAIN_ID.RULE.LESS)
        .use("style-resources-loader")
        .loader("style-resources-loader")
        .options({
          patterns: ["./src/theme/theme.less"],
        });

      chain.module
        .rule(CHAIN_ID.RULE.LESS)
        .use(CHAIN_ID.USE.LESS)
        .tap((options) => {
          options.lessOptions.math = "always";
          return options;
        });
    },
  },
});
`

执行npm run dev报错Conflict: Multiple assets emit different content to the same filename index.html
![Uploading 1123.png…]()

**Comment by wengjiacheng** [comment]
![1123](https://github.com/web-infra-dev/rsbuild/assets/24770388/df5645ab-a56e-43e1-97ac-00d0026aa1a1)

**Comment by chenjiahan** [comment]
Duplicated: https://github.com/web-infra-dev/rsbuild/issues/1225
